geoips/image_utils/maps.py

```diff
@@ -592,3 +599,9 @@ def draw_gridlines(mapobj, area_def, curr_ax, gridline_annotator, zorder=None):
     glnr.yformatter = LATITUDE_FORMATTER
     glnr.xlabel_style = {"rotation": 0}
     glnr.ylabel_style = {"rotation": 0}
+    # The following are the list of supported attributes which we can define in our
+    # gridline annotator. If specified, add these attributes to the x / y style dicts.
+    for style_arg in ["fontfamily", "fontstyle", "fontweight", "fontsize"]:
+        if style_arg in labels.keys():
+            glnr.xlabel_style[style_arg] = labels[style_arg]
+            glnr.ylabel_style[style_arg] = labels[style_arg]
```

Comment:
What happens if the values supplied aren't of the correct type or something else is wrong with the values?

Reply:
This shouldn't occur. If you look at ``geoips/schema/gridline_annotators/cartopy.yaml`` I've add type checking to ensure that the values provided for these attributes are of a certain type. Is there another way I should be doing this? More pythonic? Or is the schema type checking enough?